zc_install/includes/nginx_conf/ngx_server_template.txt

```diff
@@ -67,7 +67,7 @@
 		return 418;
 	}
 	## Allow all requests to specific static file types
-	location ~ %%store_folder%%/.+\.(md|ico|txt|html)$ {
+	location ~ %%store_folder%%/[^/]+\.(md|ico|txt|html)$ {
```

Comment:
Should this same tweak be applied everywhere in this file?

Like lines 79, 84, 89, 106, etc.